pkg/sdk/poc/generator/show_by_id_filtering.go

```diff
@@ -0,0 +1,46 @@
+package generator
+
+type ShowByIDFiltering struct {
+	Kind            string
+	Args            string
+	IdentifierBased bool
+}
+
+type ShowByIDFilteringKind uint
+
+const (
+	// Enables filtering with: Like
+	ShowByIDLikeFiltering ShowByIDFilteringKind = iota
+	// Enables filtering with: In
+	// Based on the identifier Kind
+	ShowByIDInFiltering
+	// Enables filtering with: ExtendedIn
+	// Based on the identifier Kind
+	ShowByIDExtendedInFiltering
+)
+
+func (s *Operation) withFiltering(filtering ...ShowByIDFilteringKind) *Operation {
+	for _, f := range filtering {
+		switch f {
+		case ShowByIDLikeFiltering:
+			s.ShowByIDFiltering = append(s.ShowByIDFiltering, ShowByIDFiltering{
+				Kind:            "Like",
+				Args:            "Pattern: String(id.Name())",
+				IdentifierBased: false,
+			})
+		case ShowByIDInFiltering:
```

Comment:
We can use it (non-extended In) in one object to show if it works.

Reply:
good idea, i will generate some objects to show all the filtering options

Reply:
generated `streamlits` as example, which has the simple `In` filtering